[2023-02-05 21:30:19,901] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: 06_monitoring.task_2 scheduled__2022-11-25T00:00:00+00:00 [queued]>
[2023-02-05 21:30:19,912] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: 06_monitoring.task_2 scheduled__2022-11-25T00:00:00+00:00 [queued]>
[2023-02-05 21:30:19,912] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-02-05 21:30:19,912] {taskinstance.py:1377} INFO - Starting attempt 2 of 2
[2023-02-05 21:30:19,912] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-02-05 21:30:19,946] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): task_2> on 2022-11-25 00:00:00+00:00
[2023-02-05 21:30:19,949] {standard_task_runner.py:52} INFO - Started process 35965 to run task
[2023-02-05 21:30:19,954] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', '06_monitoring', 'task_2', 'scheduled__2022-11-25T00:00:00+00:00', '--job-id', '247', '--raw', '--subdir', 'DAGS_FOLDER/06_monitoring.py', '--cfg-path', '/tmp/tmp8b9p3bs_', '--error-file', '/tmp/tmpru3u4xcm']
[2023-02-05 21:30:19,955] {standard_task_runner.py:80} INFO - Job 247: Subtask task_2
[2023-02-05 21:30:20,035] {task_command.py:371} INFO - Running <TaskInstance: 06_monitoring.task_2 scheduled__2022-11-25T00:00:00+00:00 [running]> on host pc.
[2023-02-05 21:30:20,124] {taskinstance.py:1589} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=06_monitoring
AIRFLOW_CTX_TASK_ID=task_2
AIRFLOW_CTX_EXECUTION_DATE=2022-11-25T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-11-25T00:00:00+00:00
[2023-02-05 21:30:20,126] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-02-05 21:30:20,126] {subprocess.py:74} INFO - Running command: ['bash', '-c', "sleep 2 && echo 'Second task!'"]
[2023-02-05 21:30:20,134] {subprocess.py:85} INFO - Output:
[2023-02-05 21:30:22,137] {subprocess.py:92} INFO - Second task!
[2023-02-05 21:30:22,137] {subprocess.py:96} INFO - Command exited with return code 0
[2023-02-05 21:30:22,167] {taskinstance.py:1415} INFO - Marking task as SUCCESS. dag_id=06_monitoring, task_id=task_2, execution_date=20221125T000000, start_date=20230206T023019, end_date=20230206T023022
[2023-02-05 21:30:22,216] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-02-05 21:30:22,247] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
